lib/appsignal/config.rb

```diff
@@ -127,6 +129,10 @@ def write_to_environment
       ENV['APPSIGNAL_ENABLE_MINUTELY_PROBES']       = config_hash[:enable_minutely_probes].to_s
       ENV['APPSIGNAL_HOSTNAME']                     = config_hash[:hostname].to_s
       ENV['APPSIGNAL_PROCESS_NAME']                 = $0
+
+      unless config_hash[:ca_file_path] == DEFAULT_CONFIG[:ca_file_path]
```

Comment:
Probably, but it will probably also have to be added to the agent. It looks like the agent's new http library doesn't output an error if there's a problem with the file.